internal/backend/local/test.go

```diff
@@ -272,6 +285,7 @@ type TestFileRunner struct {
 	// during the execution of a file.
 	Suite       *TestSuiteRunner
 	EvalContext *graph.EvalContext
+	Manifest    *graph.TestManifest
```

Comment:
I think this is unnecessary? The test runner has access to suite with the manifest already, so you can just access it via `runner.Suite.Manifest` directly.